run_tests.sh

```diff
@@ -4,22 +4,43 @@ cd ./human_aware_rl
 
```

Comment:
Part of me feels like we should discontinue this file and have the run-tests command be the more standard auto-test discovery that unittest provides (https://stackoverflow.com/a/15630454/5578742).

Regarding profiling, I usually just use [this decorator](https://github.com/HumanCompatibleAI/overcooked_ai/blob/df2fc6ceaa19203250aa68755b861602d8ff586b/src/overcooked_ai_py/utils.py#L206) around any function I'm currently interested in profiling, or use the built-in profiler to PyCharm (which wouldn't require having `cProfile` or `gprof2dot ` in the requirements for the repo).

That being said, if you think this set up might be useful for you in the future to be able to quickly profile tests, I'm open to keeping it :)